### PR TITLE
feat(action): add new workflow

### PR DESCRIPTION
**Pull request by warren830**
### Summary
First try to automatically upgrade helm

### Does this close any open issues?
Closes xx

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: .github/workflows/helm-upgrade.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          repository: apache/incubator-devlake
+          ref: ${{ github.ref }}
+      - name: Download helm chart repo
+        uses: actions/checkout@v2
+        with:
+          repository: apache/incubator-devlake-helm-chart
+          ref: master
+      - name: Update chart.yaml
+        run: |
+          sed -i 's/version:.*/version: ${{ github.ref }}/' charts/devlake/Chart.yaml
```
Comment (matrixji) [inline .github/workflows/helm-upgrade.yaml:22]: Need also to modify `appVersion` in Chart.yaml

File: .github/workflows/helm-upgrade.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      - name: Update values.yaml
+        run: |
+          sed -i 's/imageTag:.*/imageTag: ${{ github.ref }}/' charts/devlake/values.yaml
+      - name: Commit and Push changes
+        run: |
+          git add .
+          git commit -m "Update chart to version ${{ github.ref }}"
+          git push origin version-${{ github.ref }}
```
Comment (matrixji) [inline .github/workflows/helm-upgrade.yaml:30]: Not suggest push commits to master directly, suggest creating PR instead.

File: .github/workflows/helm-upgrade.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        with:
+          repository: apache/incubator-devlake-helm-chart
+          ref: master
+      - name: Update chart.yaml
+        run: |
+          sed -i 's/version:.*/version: ${{ github.ref }}/' charts/devlake/Chart.yaml
+      - name: Update values.yaml
+        run: |
+          sed -i 's/imageTag:.*/imageTag: ${{ github.ref }}/' charts/devlake/values.yaml
+      - name: Commit and Push changes
```
Comment (matrixji) [inline .github/workflows/helm-upgrade.yaml:26]: We could also modify `--version=xxx` in README.md

**Review by matrixji — CHANGES_REQUESTED** [review]

**Comment by warren830** [comment]
fixed by #4607

**Pull request closed without merging**